templates/telemetry/pioneer-study/pioneer-study.1.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "creationDate": {
```

Comment:
Include the common definition    
@TELEMETRY_CREATIONDATE_1_JSON@,

Reply:
fixed